cli/src/config/templates.toml

```diff
@@ -1,18 +1,18 @@
 [templates]
-branch_list = '''
+bookmark_list = '''
```

Comment:
nit: add changelog entry because this is a breaking change

Reply:
done